src/forms/checkbox/CheckBox.tsx

```diff
@@ -12,6 +12,19 @@ import * as React from "react";
 import {UID} from "react-uid";
 import {ClassNames} from "./ClassNames";
 
+/**
+ * CheckBox Props
+ * label: label of checkbox
+ * name: name of checkbox
+ * id: id of checkbox
+ * checked: if true checked else false
+ * defaultChecked: default checked value
+ * ariaLabel: arial label
+ * ariaLabelledby: arial label details
+ * disabled: property to enable disable checkbox
+ * className: css property
+ * dataqa: quality engineering testing field
```

Comment:
use below format for documentation
@param {dataqa} quality engineering tag

Reply:
Addressed